src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt

```diff
@@ -88,6 +88,7 @@ object ItemFeatures {
 
     private val headPattern =
         Regex("(?:DIAMOND|GOLD)_(?:(BONZO)|(SCARF)|(PROFESSOR)|(THORN)|(LIVID)|(SADAN)|(NECRON))_HEAD")
+    private val requirementPattern = Regex("CATACOMBS:(?<level>\\d+)")
```

Comment:
Should I do this or (it.startsWith("CATACOMBS:) it.subString(0, idk like 10).toIntOrNull()

Reply:
Regex is better IMO.

Reply:
kk will continue to use Regex in future code as well then